x-pack/modules/azure/configuration/logstash/azure.conf.erb

```diff
@@ -0,0 +1,58 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+input{
+<%=
+require 'azure_module_config_generator'
+config_generator = LogStash::Azure::ConfigGenerator.new
+config_generator.generate_input(@settings)
+%>
+}
+
+filter {
+  json {
+    source => "message"
+  }
+
+  if "_jsonparsefailure" not in [tags] {
```

Comment:
If any of those errors happen, then it should be a bug (or they pointed to a wrong event hub) and we need to adjust our parsing logic accordingly.  

I am hesitant to hide (dump/error index/drop) the error documents since it _should_ never happen.